mathesar/models.py

```diff
@@ -79,6 +79,20 @@ def name(self):
         except TypeError:
             return 'MISSING'
 
+    @property
+    def has_dependencies(self):
```

Comment:
@eito-fis Could you update this to actually check if it has dependencies? If that's going to be a lot of work, we can separate it into a new issue, but we should at least add a `TODO` here that indicates it's a placeholder and needs to be replaced.